bleak/backends/winrt/service.py

```diff
@@ -0,0 +1,46 @@
+from uuid import UUID
+from typing import List, Union
+
+from bleak.backends.service import BleakGATTService
+from bleak.backends.winrt.characteristic import BleakGATTCharacteristicWinRT
+
+from winrt.windows.devices.bluetooth.genericattributeprofile import GattDeviceService
+
+
+class BleakGATTServiceWinRT(BleakGATTService):
+    """GATT Characteristic implementation for the .NET backend, implemented with WinRT"""
+
+    def __init__(self, obj: GattDeviceService):
+        super().__init__(obj)
+        self.__characteristics = [
+            # BleakGATTCharacteristicDotNet(c) for c in obj.GetAllCharacteristics()
```

Comment:
should this comment be deleted or is it a TODO?

Reply:
Hm. Not sure. Will delete it, since it is not used. Made this change in the `dotnet` backend as well.

The getting of characteristics is async and cannot be done here in the constructor anyway, I think.